db/crud.go

```diff
@@ -2611,6 +2605,24 @@ func postWriteUpdateHLV(doc *Document, casOut uint64) *Document {
 	if doc.HLV.CurrentVersionCAS == expandMacroCASValueUint64 {
 		doc.HLV.CurrentVersionCAS = casOut
 	}
+	// backup new revision to the bucket now we have a doc assigned a CV (post macro expansion) for delta generation purposes
+	backupRev := db.deltaSyncEnabled() && db.deltaSyncRevMaxAgeSeconds() != 0
+	if db.UseXattrs() && backupRev {
```

Comment:
I think the UseXattrs() check is unnecessary here (we don't support HLV with non-xattrs)